go.mod

```diff
@@ -113,3 +114,5 @@ replace github.com/Azure/azure-sdk-for-go => github.com/Azure/azure-sdk-for-go v
 replace github.com/Azure/go-autorest => github.com/Azure/go-autorest v10.15.5+incompatible
 
 replace github.com/banzaicloud/bank-vaults => github.com/banzaicloud/bank-vaults v0.0.0-20190508130850-5673d28c46bd
+
+go 1.13
```

Comment:
no sure about this one. I don't think we officially switched to 1.13 yet. I think it would be better to hold off this until we switch jx to 1.13 as well